imaids/cassettes.py

```diff
@@ -635,10 +650,14 @@ def create_radia_object(
                         self.nr_core_blocks/2),
                     self._end_blocks_length])
                 # is_pole_list if HYBRID and first core object is POLE
+                if self.terminations_poles:
+                    end_poles = [value if i % 2 == 0 else not value for i, value in enumerate(end_poles)]
```

Comment:
It seems like the rules for ```True```/```False``` in ```_is_pole_list``` are reversed.
Here, the first block is a pole, so the last block of a period is a normal block.
Therefore, in the end termination, we should see a pole first (because the cassette ended in a normal block).